docs/3.0rc/develop/manage-states.mdx

```diff
@@ -46,26 +37,56 @@ The full list of states and state types includes:
 | Scheduled | SCHEDULED | No | The run will begin at a particular time in the future. |
```

Comment:
nit: could the state names be inline code?

## before
![image](https://github.com/user-attachments/assets/08b7d216-3dea-46b0-a15a-c8a51430a779)

## after
![image](https://github.com/user-attachments/assets/9367fe8e-d00f-4803-8f6e-ee7cdd2fbd7e)

Reply:
section courtesy of cursor if helpful
```
| Name | Type | Terminal? | Description
| --- | --- | --- | --- |
| `Scheduled` | SCHEDULED | No | The run will begin at a particular time in the future. |
| `Late` | SCHEDULED | No | The run's scheduled start time has passed, but it has not transitioned to PENDING (15 seconds by default). |
| <span class="no-wrap">`AwaitingRetry`</span> | SCHEDULED | No | The run did not complete successfully because of a code issue and had remaining retry attempts. |
| `Pending` | PENDING | No | The run has been submitted to execute, but is waiting on necessary preconditions to be satisfied. |
| `Running` | RUNNING | No | The run code is currently executing. |
| `Retrying` | RUNNING | No | The run code is currently executing after previously not completing successfully. |
| `Paused` | PAUSED | No | The run code has stopped executing until it receives manual approval to proceed. |
| `Cancelling` | CANCELLING | No | The infrastructure on which the code was running is being cleaned up. |
| `Cancelled` | CANCELLED | Yes | The run did not complete because a user determined that it should not. |
| `Completed` | COMPLETED | Yes | The run completed successfully. |
| `Cached` | COMPLETED | Yes | The run result was loaded from a previously cached value. |
| `RolledBack` | COMPLETED | Yes | The run completed successfully but the transaction rolled back and executed rollback hooks. |
| `Failed` | FAILED | Yes | The run did not complete because of a code issue and had no remaining retry attempts. |
| `Crashed` | CRASHED | Yes | The run did not complete because of an infrastructure issue. |
```

Reply:
@zzstoatzz i'm not opposed to it but it seems arbitrary given that both the name and type are code references -- would it look OK if I inlined both name and type in the table?

Reply:
for me it was just because thats how people will see state names in the UI, but yeah that's also fine with me